### PR TITLE
Add configuration documents for Monitoring

### PR DESCRIPTION
**Pull request by kazydek**
**Description**

Changes proposed in this pull request:

- Created configuration docs for these Monitoring sub-charts:
    -  [`alertmanager`](https://github.com/kyma-project/kyma/blob/master/resources/monitoring/charts/alertmanager)
   - [`grafana`](https://github.com/kyma-project/kyma/tree/master/resources/monitoring/charts/grafana)
   - [`prometheus`](https://github.com/kyma-project/kyma/blob/master/resources/monitoring/charts/prometheus)

**Related issue(s)**
See also #4080

### CODE REVIEW

File: docs/monitoring/05-01-alertmanager.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+>* [Sub-charts overrides](/root/kyma/#configuration-helm-overrides-for-kyma-installation-sub-chart-overrides)
+
+## Configurable parameters
+
+This table lists the configurable parameters, their descriptions, and default values:
+
+| Parameter | Description | Default value |
+|-----------|-------------|---------------|
+| **global.alertTools.credentials.slack.apiurl** | Specifies the URL endpoint which sends the alerts fired by Prometheus rules.  | None |
+| **global.alertTools.credentials.slack.channel** | Refers to the Slack channel which receives the alerts notifications. | None |
```
Comment (klaudiagrz) [inline docs/monitoring/05-01-alertmanager.md]:
```suggestion
| **global.alertTools.credentials.slack.channel** | Refers to the Slack channel which receives alert notifications. | None |
```

File: docs/monitoring/05-01-alertmanager.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+>* [Helm overrides for Kyma installation](/root/kyma/#configuration-helm-overrides-for-kyma-installation)
+>* [Sub-charts overrides](/root/kyma/#configuration-helm-overrides-for-kyma-installation-sub-chart-overrides)
+
+## Configurable parameters
+
+This table lists the configurable parameters, their descriptions, and default values:
+
+| Parameter | Description | Default value |
+|-----------|-------------|---------------|
+| **global.alertTools.credentials.slack.apiurl** | Specifies the URL endpoint which sends the alerts fired by Prometheus rules.  | None |
```
Comment (klaudiagrz) [inline docs/monitoring/05-01-alertmanager.md]:
```suggestion
| **global.alertTools.credentials.slack.apiurl** | Specifies the URL endpoint which sends alerts fired by Prometheus rules.  | None |
```

File: docs/monitoring/05-01-alertmanager.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+title: Alertmanager sub-chart
```
Comment (a-thaler) [inline docs/monitoring/05-01-alertmanager.md]: WHy it is relevant that there is a sub-chart? Isn't that confusing for the end-user? WHat value does he have from that info?

File: docs/monitoring/05-01-alertmanager.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+To configure the Alertmanager sub-chart, override the default values of its `values.yaml` file. This document describes parameters that you can configure.
+
+>**TIP:** To learn more about how to use overrides in Kyma, see the following documents:
+>* [Helm overrides for Kyma installation](/root/kyma/#configuration-helm-overrides-for-kyma-installation)
+>* [Sub-charts overrides](/root/kyma/#configuration-helm-overrides-for-kyma-installation-sub-chart-overrides)
+
+## Configurable parameters
+
+This table lists the configurable parameters, their descriptions, and default values:
+
```
Comment (a-thaler) [inline docs/monitoring/05-01-alertmanager.md]: that configuration options should be put into a secret, not a configmap as they contain secrets. That needs to be mentioned somehow

File: docs/monitoring/05-01-alertmanager.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+| Parameter | Description | Default value |
+|-----------|-------------|---------------|
+| **global.alertTools.credentials.slack.apiurl** | Specifies the URL endpoint which sends alerts triggered by Prometheus rules.  | None |
+| **global.alertTools.credentials.slack.channel** | Refers to the Slack channel which receives notifications on new alerts. | None |
+| **global.alertTools.credentials.victorOps.routingkey** | Defines the team routing key in VictorOps. | None |
```
Comment (a-thaler) [inline docs/monitoring/05-01-alertmanager.md]: maybe point to victorOps docu

File: docs/monitoring/05-01-alertmanager.md (hunk 3 above)
Comment (kazydek) [reply docs/monitoring/05-01-alertmanager.md]: We are following the template here and we distinguish charts from the sub-charts for the users to easily locate those values.

Comment (a-thaler) [reply docs/monitoring/05-01-alertmanager.md]: ok

**Pull request merged**